views/fe-utils/fetchUtil.js

```diff
@@ -214,6 +214,19 @@ const fetchGraphTableCSV = async (chart_id, filters, filename) => {
   return res
 }
 
+const updateUserAccess = async (user) => {
```

Comment:
the `user` value here isn't a `user` but the updated user values. Let's split out the uid from the attributes we want to update (suggestion below).

```suggestion
const updateUserAccess = async (uid, attributes) => {
```

Reply:
[b780b02](https://github.com/AMP-SCZ/dpdash/pull/294/commits/b780b02b79e59c25ddadb4cd5634c5547e9dbda7)